debug-init-env.sh

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
```

Comment:
@pfrolke : *instead of* changing the paths in this script, move the script to the command sub-module. The `run-*.sh` scripts will be run from that submodule.